admin_manual/maintenance/migrating.rst

```diff
@@ -29,3 +29,114 @@ To start, let us be specific about the use case. A configured ownCloud instance
 
 #.  Change the ``CNAME`` entry in the DNS to point your users to the new
     location.
+
+
+
+=================
+Example migration
+=================
+
+.. note:: For this example to work, you need **ssh** on both servers and the **PermitRootLogin** to be set to "yes" on the new server.
+
+Install SSH::
+
+   apt install openssh-server openssh-client -y
+
+Edit SSH-Config::
+
+   nano /etc/ssh/sshd_config
+
+Change PermitRootLogin to yes::
+
+   PermitRootLogin yes
+
+=========
+Migration
+=========
+
+1. Install ownCloud on new server
+
+2. Put original server in maintenance mode:
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+
+Switch to maintenance mode::
+
+      sudo -u www-data php occ maintenance:mode --on
+
+
+wait for 6-7 min and stop apache2::
+
+   service apache2 stop
+
+3. Export and Import the database
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+Export on original server
+
+SYNOPSIS::
+
+   mysqldump --lock-tables -h [server] -u [username] -p[password] [db_name] > owncloud-dbbackup_`date +"%Y%m%d"`.bak
+
+Example::
+
+   mysqldump --lock-tables -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
+
+.. note:: (just a reminder) Parameters you need for the command above were set when creating database for ownCloud
+
+::
+
+   CREATE DATABASE IF NOT EXISTS owncloud;
+   GRANT ALL PRIVILEGES ON owncloud.* TO 'username'@'localhost' IDENTIFIED BY 'password';
+
+Export command::
+
+   rsync -Aaxt owncloud-dbbackup.bak root@new_server_address:/var/www/owncloud 
+
+Import on new server::
+
+   mysql -h localhost -u admin -ppassword owncloud < owncloud-dbbackup.bak
+
+4. Copy data, config to new server::
+
+      rsync -Aaxt config data root@new_server_address:/var/www/owncloud 
+
+.. warning:: You must keep the data/ directory’s original filepath. Do not change this!
```

Comment:
I propose to to reword:
``In case you also want to move your datadirectory to another location at the target server, it is advised to do this as a second step. Please see the datadirectory migration document <link> for more details.``